ledger/src/ui/src/signer_authorization.c

```diff
@@ -50,7 +51,7 @@ static const uint16_t INITIAL_SIGNER_ITERATION = PARAM_INITIAL_SIGNER_ITERATION;
 #endif
 
 // Current signer status
-static const sigaut_signer_status_t N_current_signer_status_var;
+static NON_VOLATILE sigaut_signer_status_t N_current_signer_status_var;
```

Comment:
We could potentially remove the `static` in order to be able to use `extern` from unit tests. I don't see a lot of harm in it, do you?